python/pyarrow/array.pxi

```diff
@@ -921,81 +921,15 @@ cdef class Array(_PandasConvertible):
 
     def take(self, object indices):
         """
-        Take elements from an array.
-
-        The resulting array will be of the same type as the input array, with
-        elements taken from the input array at the given indices. If an index
-        is null then the taken element will be null.
```

Comment:
Can you keep this "If an index is null then the taken element will be null." sentence in the docstring in the compute module?

Reply:
and maybe copy paste the example below as well